x-pack/plugins/security_solution/public/management/pages/event_filters/view/event_filters_list_page.tsx

```diff
@@ -236,11 +236,11 @@ export const EventFiltersListPage = memo(() => {
           <EuiText color="subdued" size="xs" data-test-subj="eventFiltersCountLabel">
             <FormattedMessage
               id="xpack.securitySolution.eventFilters.list.totalCount"
-              defaultMessage="{total, plural, one {# event filter} other {# event filters}}"
+              defaultMessage="Showing {total, plural, one {# event filter} other {# event filters}}"
```

Comment:
if you change this defaultMessage make sure to remove the existing translations otherwise they won't be updated.

Reply:
i think i was able to change the other translations? i believe i18n would have failed if there we're conflicting translations

Reply:
Yes, but this is not the case of conflicting translations, rather an old string being updated and the old translation still on place.

Reply:
@academo not sure I understand either.
The update to the strings translations are not done by us. So if you change the default values you should not encounter i18n errors. It's only when you delete keys or changes the key values, in those cases the i18n checker detects that there are keys that are no longer being used by the application in the translated files, and thus we need to run it locally to remove them.

Reply:
@paul-tavares @parkiino what I mean is not that you are creating a conflicting translation but an incorrect translation.

For example. A translation says "This is a text". and someone translated in Portuguese to "Este é um texto". Then you change the original `defaultMessage` to "This is a message" but you don't delete the old existing translation. The portuguese translation now displays (in that language) a message that doesn't longer correspond to the text in English.

In this particular case, the translation changed removing some words but for JP and ZH translations may still contain that word.

Perhaps I don't understand well how the translation system works and translators get a notification when an original text changes or it gets automatically removed?

Reply:
Yeah, I'm not sure about that you are assuming here. We don't normally have to worry about translations (which Elastic only provides 2 I think out of the box).